frontend/packages/kubevirt-plugin/src/components/vms/vm-details-page.tsx

```diff
@@ -74,14 +73,33 @@ export const VirtualMachinesDetailsPage: React.FC<VirtualMachinesDetailsPageProp
     disksPage,
   ];
 
+  const resources = [
+    getResource(VirtualMachineInstanceModel, {
+      name,
+      namespace,
+      isList: false,
+      prop: 'vmi',
+      optional: true,
+    }),
+    getResource(PodModel, { namespace, prop: 'pods' }),
+    getResource(VirtualMachineInstanceMigrationModel, { namespace, prop: 'migrations' }),
+  ];
+
   return (
-    <DetailsPage {...props} menuActions={menuActionsCreator} pages={pages} resources={resources} />
+    <DetailsPage
+      {...props}
+      name={name}
+      namespace={namespace}
+      kind={VirtualMachineModel.kind}
+      kindObj={VirtualMachineModel}
+      menuActions={menuActionsCreator}
```

Comment:
`menuActions` need to be fixed to take VMI into account + also the Consoles

Reply:
nice !